### PR TITLE
Updating the playbook for mounting error

### PR DESCRIPTION
**Pull request by Tulsishah**

### CODE REVIEW

File: docs/playbook.md [hunk 1]
```diff
@@ -38,5 +38,25 @@ Enter NO_PUBKEY here (eg: 6494C6D6997C215E is NO_PUBKEY for above error)
...
+
+While running
+
+* gcsfuse --implicit-dirs <bucket> ~/gcsfuse
+
+Err: daemonize.Run: readFromProcess: sub-process: mountWithArgs: mountWithConn: Mount: mount: running /usr/bin/fusermount3: exit status 1
+
+**Reason**
+
+If bucket is already mounted in that folder and we try to mount it again in the same folder without un-mounting it, It will show this error.
```
Comment (vadlakondaswetha) [inline docs/playbook.md:54]: small i

File: docs/playbook.md [hunk 2]
```diff
@@ -38,5 +38,25 @@ Enter NO_PUBKEY here (eg: 6494C6D6997C215E is NO_PUBKEY for above error)
 
 **Solution**
 
-*   sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6494C6D6997C215E
-*   sudo apt-get update
\ No newline at end of file
+* sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 6494C6D6997C215E
+* sudo apt-get update
+
+### Mounting Bucket doesn't get Succeeded
```
Comment (vadlakondaswetha) [inline docs/playbook.md:44]: please add this under mounting section.

**Review by vadlakondaswetha — CHANGES_REQUESTED** [review]

File: docs/playbook.md (hunk 1 above)
Comment (Tulsishah) [reply docs/playbook.md]: fixed

File: docs/playbook.md (hunk 2 above)
Comment (Tulsishah) [reply docs/playbook.md]: fixed

File: docs/playbook.md [hunk 3]
```diff
@@ -18,6 +18,26 @@ Please consult the [flags.go](https://github.com/GoogleCloudPlatform/gcsfuse/blo
 ### Mount successful but files not visible
 This is explained in the [semantics](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/semantics.md#implicit-directories) page and can possibly be fixed by passing the --implicit-dirs=true flag to the gcsfuse command.
 
+### Mounting Bucket doesn't get Succeeded
```
Comment (vadlakondaswetha) [inline docs/playbook.md:21]: Do you want to keep the title as mount failed with fusermount3 exit status 1 to keep in sync with other titles here.

File: docs/playbook.md [hunk 4]
```diff
@@ -18,6 +18,26 @@ Please consult the [flags.go](https://github.com/GoogleCloudPlatform/gcsfuse/blo
...
+
+While running
+
+* gcsfuse --implicit-dirs <bucket> ~/gcsfuse
```
Comment (vadlakondaswetha) [inline docs/playbook.md:25]: also this is not required. mounting heading itself means you are invoking gcsfuse mount command. Just mention as follows

The error comes when bucket is already mounted in a folder and we try to mount it again in the same folder without un-mounting it. Fix it to first umount and then remount. Command for un mounting. 
sudo umount ~/gcsfuse

File: docs/playbook.md (hunk 3 above)
Comment (Tulsishah) [reply docs/playbook.md]: fixed

File: docs/playbook.md [hunk 5]
```diff
@@ -18,6 +18,11 @@ Please consult the [flags.go](https://github.com/GoogleCloudPlatform/gcsfuse/blo
 ### Mount successful but files not visible
 This is explained in the [semantics](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/docs/semantics.md#implicit-directories) page and can possibly be fixed by passing the --implicit-dirs=true flag to the gcsfuse command.
 
+### Mount failed with fusermount3 exit status 1
+
```
Comment (vadlakondaswetha) [inline docs/playbook.md:22]: there is no new line between header and description for others. please remove it here.

File: docs/playbook.md [hunk 6]
```diff
@@ -18,6 +18,11 @@ Please consult the [flags.go](https://github.com/GoogleCloudPlatform/gcsfuse/blo
...
+The error comes when bucket is already mounted in a folder and we try to mount it again in the same folder without un-mounting it. Fix it to first umount and then remount. Command for un-mounting.
+sudo umount ~/gcsfuse
```
Comment (vadlakondaswetha) [inline docs/playbook.md:24]: instead you should say sudo umount <mounted-dir-name>

**Review by vadlakondaswetha — CHANGES_REQUESTED** [review]

File: docs/playbook.md (hunk 5 above)
Comment (Tulsishah) [reply docs/playbook.md]: fixed

File: docs/playbook.md (hunk 6 above)
Comment (Tulsishah) [reply docs/playbook.md]: fixed

Comment (vadlakondaswetha) [reply docs/playbook.md]: sorry my bad. comment was incomplete. you should mention sudo umount <mounted-dir-name>

**Review by vadlakondaswetha — APPROVED** [review]

**Pull request merged**